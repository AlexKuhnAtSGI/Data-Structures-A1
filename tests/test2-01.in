not
whole
not
wholly
separate
but
syncopated