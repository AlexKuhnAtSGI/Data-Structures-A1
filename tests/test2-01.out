not
not
separate
syncopated
whole
wholly
but
